squiggy/routes.py

```diff
@@ -101,4 +102,9 @@ def after_api_request(response):
 
 def _user_loader(user_id=None):
     from squiggy.lib.login_session import LoginSession
+    user_id = user_id or _lti_launch()
```

Comment:
There is no performance cost in having `_lti_launch()` as a fallback during flask-login's `load_user` step.  If the user has an active session then the method is not invoked.